src/utils/index.ts

```diff
@@ -72,12 +72,39 @@ export function validateEmail(email: unknown): email is string {
 }
 
 export function validateURL(url: string) {
-  const expression = /(https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|www\.[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9]+\.[^\s]{2,}|www\.[a-zA-Z0-9]+\.[^\s]{2,})/
+  const expression =
+    /(https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|www\.[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9]+\.[^\s]{2,}|www\.[a-zA-Z0-9]+\.[^\s]{2,})/;
 
-  if (url.match(expression)){
-    return true
+  if (url.match(expression)) {
+    return true;
   }
 }
+
+export function slugify(title: string) {
+  return title
+    .normalize("NFKD")
+    .toLocaleLowerCase()
+    .trim()
+    .replace(/\s+/g, "-")
+    .replace(/[^\w\-]+/g, "")
+    .replace(/\_/g, "-")
+    .replace(/\-\-+/g, "-")
+    .replace(/\-$/g, "");
+}
+
+export function randomCharacters() {
+  let characters = "abcdefghijklmnopqrstuvwxyz0123456789";
+
+  let randomString = "";
+
+  for (let i = 0; i < 6; i++)
+    randomString += characters.charAt(
+      Math.floor(Math.random() * characters.length)
+    );
```

Comment:
It would be nice to put curly braces to this for loop, since most of the `if` and `for` statements using curly braces even if they're one-liner, to make sure we're consistent.